### ISSUE TITLE
[Docs]Clarify alignment of suggestion in docs

### ISSUE THREAD

**Issue by udcode** [maheshmnj/searchfield#90]
**Is your feature request related to a problem? Please describe.**
need to align RTL texts from right to left currently the suggestion is sticked to the left side

**Describe the solution you'd like**
Add Alignment argument to the searchfield widget

**Describe alternatives you've considered** 

**Additional context**
Ill add a PR for the solution

**Comment by maheshmnj** [comment]
Hello @udcode, thanks for filing the issue. This is already possible as of today  by passing a child property to 
`SearchFieldListItem` e.g see this answer https://github.com/maheshmnj/searchfield/issues/40#issuecomment-1208920478

```dart
 SearchField(
      suggestions: _statesOfIndia
      .map((e) => SearchFieldListItem(e,
          child: Align(
              alignment: Alignment.centerRight,
              child: Text(e,
                  style: TextStyle(color: Colors.red),
              ),
          )
        ).toList(),
...
)
```
there have been similar issue reports in the past  

https://github.com/maheshmnj/searchfield/issues/66
https://github.com/maheshmnj/searchfield/issues/40

**Comment by maheshmnj** [comment]
So I believe we should just clarify this in the readme/docs with an example and it should resolve.

**Comment by maheshmnj** [comment]
Fixed in [Release v0.8.4](https://pub.dev/packages/searchfield/changelog)
Thank you for your contribution!

cc: @udcode
